src/types.ts

```diff
@@ -12,18 +11,13 @@ export interface Question {
   max_char_limit: number | null;
 }
 
-export interface Item {
-  type: itemType;
-  details: Question;
-}
-
 export interface IconButtonTitleConfig {
   value: string;
-  class: string;
+  class?: string;
```

Comment:
`?` means nullable, I'm assuming. What is the use case with `value` is present, but `class` is not?

Reply:
yes, the use case here is that most of our buttons might have the same styles - specified in class. 
Now, I don't want to keep repeating all the classes, just want to send the right text for a given button - specified by `value`. Doing this lets me provide just the text of the button while letting me use default values for the `class` which I can override if I want, but in a lot of cases, the default one just works.

Reply:
Wokay!